rero_ils/modules/patrons/views.py

```diff
@@ -148,15 +150,19 @@ def logged_user():
 )
 def profile(viewcode):
     """Patron Profile Page."""
-    tab = 'checkouts'
+    tab = request.args.get('tab', 'loans')
+    if tab not in ['loans', 'requests', 'fees', 'history']:
+        abort(400)
     patron = Patron.get_patron_by_user(current_user)
     if patron is None:
         raise NotFound()
+    if 'patron' not in patron.get('roles'):
```

Comment:
better to use `patron.is_patron` to be more generic

Reply:
done